pkg/scheduler/framework/plugins/noderesources/least_allocated.go

```diff
@@ -63,25 +64,58 @@ func (la *LeastAllocated) ScoreExtensions() framework.ScoreExtensions {
 }
 
 // NewLeastAllocated initializes a new plugin and returns it.
-func NewLeastAllocated(_ runtime.Object, h framework.FrameworkHandle) (framework.Plugin, error) {
+// resource without the weight defined or explicitly defined as zero will be applied with defined weight, i.e. "1" instead,
+// if the resource is not accountable for the final score, just remove the resource from the plugin args.
+func NewLeastAllocated(laArgs runtime.Object, h framework.FrameworkHandle) (framework.Plugin, error) {
+	args, err := getNodeResourcesLeastAllocatedArgs(laArgs)
+	if err != nil {
+		return nil, err
+	}
+
+	resToWeightMap := make(resourceToWeightMap)
+	if len(args.Resources) == 0 {
+		// If no resources specified, used the default set.
+		resToWeightMap = defaultRequestedRatioResources
```

Comment:
Put this in `apis/config/v1alpha2/defaults.go`?

Reply:
done